src/praisonai-agents/tests/test_parallel_tools.py

```
@@ -0,0 +1,197 @@
+    print(f"\nPrompt: {prompt}")
+    
+    # Test sequential agent (baseline)
+    print("\n--- Sequential Agent ---")
+    sequential_start = time.time()
+    try:
+        sequential_result = sequential_agent.start(prompt)
+        sequential_time = time.time() - sequential_start
+        print(f"Sequential agent completed in: {sequential_time:.2f}s")
+        print(f"Result length: {len(sequential_result)} chars")
+        print(f"Result preview: {sequential_result[:200]}...")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)